build

```diff
@@ -31,7 +31,8 @@ echo "Building ${NAME}..."
 go clean -help 2>&1 | grep -F '[-cache]' >/dev/null && go clean -cache -testcache internal
 go build -buildmode=pie -ldflags "${GLDFLAGS}" -o ${BIN_PATH}/${NAME} ${REPO_PATH}/internal
 
-NAME="ignition-validate"
-
-echo "Building ${NAME}..."
-go build -ldflags "${GLDFLAGS}" -o ${BIN_PATH}/${NAME} ${REPO_PATH}/validate
+for bin in validate liveapply; do
```

Comment:
One downside of a new binary is it's going to basically be another one of these right?
`-rwxr-xr-x. 2 root root 22M Jan  1  1970 /usr/lib/dracut/modules.d/30ignition/ignition`

I think perhaps we should have a tiny shell script like:
```
#!/usr/bin/bash
exec /usr/lib/dracut/modules.d/30ignition/ignition live-apply
```
as `/usr/sbin/ignition-live-apply` or so?

Reply:
Hmm, do we actually want this in the OS though? I wasn't planning to push for that. For CoreOS layering we could use it as part of a multi-stage build. In which case having it be independent of the main Ignition binary would be nicer.